packages/astro/src/vite-plugin-astro-server/server-state.ts

```diff
@@ -0,0 +1,54 @@
+import type { ManifestData, RouteData } from '../@types/astro';
+
+export type ErrorState = 'fresh' | 'error' | 'unresponsive';
```

Comment:
Not seeing where `unresponsive` is being used... remove?

Reply:
It is unused for now. I'll remove it if it's not used by the time this is ready for review.